policyengine_us/variables/gov/states/me/adjusted_gross_income/me_pension_income_deduction.py

```diff
@@ -0,0 +1,54 @@
+from policyengine_us.model_api import *
+
+
+class me_pension_income_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "ME AGI pension income subtractions"
+    unit = USD
+    documentation = (
+        "ME pension income subtractions, part of ME agi subtractions."
+    )
+    definition_period = YEAR
+    defined_for = StateCode.ME
+    dict(
```

Comment:
I don't this we support dict references at the moment (also needs the `reference = `)
```suggestion
    reference = "https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_1s_ff.pdf"
```